### ISSUE TITLE
Incorrect Bright Star Association for ATLAS/Lasair object

### ISSUE THREAD

**Issue by mFulton07** [thespacedoctor/sherlock#155]
This ATLAS object: https://star.pst.qub.ac.uk/sne/atlas4/candidate/1192044031465256100/ which is also in Lasair: https://lasair-ztf.lsst.ac.uk/object/ZTF22aasoali/ has been classified as a bright star by Sherlock. 

This is not right. Should have been flagged as an Orphan. There are no bright sources nearby. There is a galaxy crossmatch also, but it is ~80 arcsecs away. The object has been classified as a Type IIn: https://www.wis-tns.org/object/2022rhl.

**Comment by thespacedoctor** [comment]
The bright star reported is in the bottom left of this image (the transient location is the red circle).

<img width="152" alt="image" src="https://user-images.githubusercontent.com/3385557/189917283-9d2e020e-72c1-428c-8307-9b22fae72acc.png">

@genghisken is there a way to view all ATLAS objects associated with classified TNS sources? It would be good to quantify these Sherlock-triggered misses to see how common they are. I could also run Sherlock directly against the coordinates of classified TNS sources and see what results I get.

**Comment by smarttgit** [comment]
Seems too far from the transient to be classified as a BS ? Maybe we should reduce that radius. Agree that a TNS match may tell us what we have missed, that's a good test Dave. 

Also important is sherlock's text statement which doesn't seem to be coherent. A 16.3 mag galaxy but only in PS1 ? And this possible host seems too far from the transient to be associated. 

Classified as BS, at 88.46 arcsec.
Best crossmatch is galaxy
The transient is possibly associated with 164242902141422599; an r=16.32 mag galaxy found in the PS1 catalogue. Its located 48.63" N, 73.89" W from the galaxy centre.

**Comment by smarttgit** [comment]
All seems strangely wrong, the marshall seems to point at a start, not a PS1 galaxy, and the blue square on Lasair does not have an object.
https://www.pessto.org/marshall/transients/search?q=2022rhl
Think this is an edge case that has mixed up information (perhaps bogus info in the catalogues) and the usual algorithm gets caught out ?

**Comment by thespacedoctor** [comment]
Agreed I may need to revisit the BS radius, but I think this is an edge case.

The BS to the bottom left *is* identified as a galaxy in PS1, but Sherlock treats all 'galaxies' above a given brightness as a BS ... and I think it was correct to do so in this case. Saturated stars mimic the morphology of galaxies and trip up the source classification machine learning algorithms.

> Best crossmatch is galaxy

This is generated by the Lasair code, not Sherlock, and is misleading. It doesn't appear in the new pages ;)

**Comment by mFulton07** [comment]
Although this galaxy is not known to _just_ Pan-STARRS, it is also in 2MASS/2MASX according to NED and has a photometric redshift (PohotZ~ 0.053211).

**Comment by genghisken** [comment]
I can pull out any TNS crossmatches in the stars list (currently 48 million) - restricted to (currently 21 million) BS objects if necessary, but I might as well do them all.  Might take a while!

**Comment by thespacedoctor** [comment]
Don't worry Ken, the direct download of coordinates from TNS will be quicker and I will be able to run the latest version of sherlock against them. The ATLAS Sherlock classifications could be from a few different versions.
